index.html

```diff
@@ -62,164 +63,166 @@ <h5 class="subtitle">LET'S CONNECT</h5>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">Back End Dev</p>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">2015</p>
+            </div>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work-tags-div disable-default">
+              <li>
+                <p class="work-tags">html</p>
+              </li>
+              <li>
+                <p class="work-tags">css</p>
+              </li>
+              <li>
+                <p class="work-tags">javascript</p>
+              </li>
+            </ul>
+            <button outlined class="btn work-button">See Project</button>
           </div>
-          <div class="work-info">
-            <p class="work-info-title">CANOPY</p>
-            <span class="grey-dot"></span>
-            <p class="work-info-sub">Back End Dev</p>
-            <span class="grey-dot"></span>
-            <p class="work-info-sub">2015</p>
+        </div>
+        <div class="work-card work-card-reversed">
+          <img class="work-img" src="assets/sample-4.jpg" />
+          <div class="work-content">
+            <h1 class="work-headline">Cities</h1>
+            <div class="work-info">
+              <p class="work-info-title">CANOPY</p>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">Back End Dev</p>
+              <span class="grey-dot"></span>
+              <p class="work-info-sub">2015</p>
+            </div>
+            <p class="work-description">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="work-tags-div disable-default">
+              <li>
+                <p class="work-tags">html</p>
+              </li>
+              <li>
+                <p class="work-tags">css</p>
+              </li>
+              <li>
+                <p class="work-tags">javascript</p>
+              </li>
+            </ul>
+            <button outlined class="btn work-button">See Project</button>
           </div>
-          <p class="work-description">
-            A daily selection of privately personalized reads; no accounts or
-            sign-ups required.
+        </div>
+      </div>
+    </section>
+    <section class="container top-right-curved">
+      <div class="about left-right-margin">
+        <div class="about-myself">
+          <h1 class="about-headline">
+            About <br />
+            Myself
+          </h1>
+          <p class="about-description">
+            Hello I’m a software developer! I can help you build a product ,
+            feature or website Look through some of my work and experience! If
+            you like what you see and have a project you need coded, don’t
+            hestiate to contact me.
```

Comment:
- [ ] You should stick to the design as much as possible. Please use ` color: rgba(52, 69, 99, 1);` for you p tag to match the design